src/components/NcBreadcrumb/NcBreadcrumb.vue

```diff
@@ -137,13 +140,21 @@ export default {
 		},
 
 		/**
-		 * Set a css icon-class to show an icon instead of the name text.
+		 * Set a css icon-class to show an icon along name text (if iconText is provided, otherwise just icon).
 		 */
 		icon: {
 			type: String,
 			default: '',
 		},
 
+		/**
+		 * Enable name to accompany the icon to create icon + text NcButton
+		 */
+		iconText: {
```

Comment:
```suggestion
		forceIconText: {
```
`iconText` makes it seem like a string value when it is actually a boolean and `name` is the actual text that is displayed

Reply:
Agreed, that conveys the props usage better 👍